### PR TITLE
first version of `schur_index`

### PR DESCRIPTION
**Pull request by ThomasBreuer**
This is (almost) what I need at the moment.

Later we will also want Schur indices over p-adic fields and number fields, and also computations using subgroups if the character table knows a group.

### CODE REVIEW

File: src/Groups/group_characters.jl [hunk 1]
```diff
@@ -600,6 +626,14 @@ function Base.:^(chi::GAPGroupClassFunction, n::IntegerUnion)
     return GAPGroupClassFunction(chi.table, chi.values ^ n)
 end
 
+function Base.:^(chi::GAPGroupClassFunction, tbl::GAPGroupCharacterTable)
+    return GAPGroupClassFunction(tbl, GAP.Globals.InducedClassFunction(chi.values, tbl.GAPTable))
+end
+
+function complex_conjugate(chi::GAPGroupClassFunction)
```
Comment (thofma) [inline src/Groups/group_characters.jl]:
```suggestion
function conj(chi::GAPGroupClassFunction)
```
This is the name for complex conjugate so far. We should try to be consistent here. We can add the alias later.

Comment (ThomasBreuer) [reply src/Groups/group_characters.jl]: @thofma Thanks. I will change the name.
(However, I think that `conj` is a bad name. In particular in contexts where "algebraic conjugacy" occurs, and complex conjugation is just a special case.)

Comment (fingolfin) [reply src/Groups/group_characters.jl]: Agreed, but it's a name already defined by `Julia`, so we are adding methods to it... but we certainly could have alias names.

File: src/Groups/group_characters.jl [hunk 2]
```diff
@@ -676,3 +736,79 @@ function character_field(chi::GAPGroupClassFunction)
...
+      for i in 1:length(cand)
+        for j in 1:i
+          bound = gcd(bound, scalar_product(chi, cand[i] * cand[j]))
+          bound == 1 && return 1
+        end
+      end
+    end
+
+    # For the moment, we do not have more character theoretic criteria.
+    return
```
Comment (fingolfin) [inline src/Groups/group_characters.jl:813]: Matter of taste, but I'd usually prefer to be explicit about showing that we return `nothing` here, intentionally (as opposed to having forgotten to provide a return value). Feel free to ignore this!
```suggestion
    return nothing
```

File: src/Groups/group_characters.jl [hunk 3]
```diff
@@ -612,6 +646,32 @@ function(chi::GAPGroupClassFunction)(g::BasicGAPGroupElem)
...
+Return the `n`-th Frobenius-Schur indicator of `chi`, that is,
+the value $(∑_{g ∈ G} chi(g^n))/|G|$, where $G$ is the group of `chi`.
+
+If `chi` is irreducible then `indicator(chi)` is
+`0` if `chi` is not real-valued,
+`1` if `chi` is afforded by a real representation of $G$, and
+`-1` if `chi` is real-valued but not afforded by a real representation of $G$.
+"""
+function indicator(chi::GAPGroupClassFunction, n::Int = 2)::Int
+    return GAP.Globals.Indicator(chi.table.GAPTable, GAP.GapObj([chi.values]), n)[1]
```
Comment (fingolfin) [inline src/Groups/group_characters.jl:672]:
```suggestion
function indicator(chi::GAPGroupClassFunction, n::Int = 2)
    return GAP.Globals.Indicator(chi.table.GAPTable, GAP.GapObj([chi.values]), n)[1]::Int
```
The code you have right now indicates that the return value shall be *converted* to `Int`, via `convert(Int, ...)`; while my suggestion is just a type assertion, which may be slightly cheaper.

**Review by fingolfin — APPROVED** [review]
Fine by me, two minor optional suggestions

File: src/Groups/group_characters.jl (hunk 2 above)
Comment (ThomasBreuer) [reply src/Groups/group_characters.jl:813]: Fine.
(I had written `return` in order to make explicit that the return statement has not been forgotten.)

File: src/Groups/group_characters.jl (hunk 3 above)
Comment (ThomasBreuer) [reply src/Groups/group_characters.jl:672]: O.k.

**Pull request merged**